comp/trace/bundle_mock.go

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package core implements the "core" bundle, providing services common to all
+// agent flavors and binaries.
+//
+// The constituent components serve as utilities and are mostly independent of
+// one another.  Other components should depend on any components they need.
+//
+// This bundle does not depend on any other bundles.
+
+//go:build test
+
+package trace
+
+import (
+	coreconfig "github.com/DataDog/datadog-agent/comp/core/config"
+	"github.com/DataDog/datadog-agent/comp/trace/config"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+// team: agent-apm
+
+// MockBundle defines the fx options for this bundle.
+var MockBundle = fxutil.Bundle(
```

Comment:
Bundles should contain only internal components. The reason is that if 2 bundles use `config.MockModule` then they cannot be used together.